azurecompute-arm/src/main/java/org/jclouds/azurecomputearm/compute/AzureComputeServiceAdapter.java

```diff
@@ -260,9 +260,12 @@ public boolean apply(final OSImage input) {
                               image.eula());
    }
 
+   private String getSubscriptionId() {
```

Comment:
We should not try to access test parameters from non-test code. It is better to have todo there as that class needs complete rewrite anyway.